src/main/java/de/siphalor/amecs/impl/mixin/MixinKeyBinding.java

```diff
@@ -124,9 +126,10 @@ private static void onKeyPressed(InputUtil.Key keyCode, CallbackInfo callbackInf
 		callbackInfo.cancel();
 	}
 
-	@Inject(method = "setKeyPressed", at = @At("HEAD"))
+	@Inject(method = "setKeyPressed", at = @At("HEAD"), cancellable = true)
 	private static void setKeyPressed(InputUtil.Key keyCode, boolean pressed, CallbackInfo callbackInfo) {
 		KeyBindingManager.setKeyPressed(keyCode, pressed);
+		callbackInfo.cancel();
```

Comment:
> 
> 
> I haven't looked at the code again: So just asking if this is really safe to cancel.
> 
> I'd like to cancel as rare as possible.

I understand that. The cancel it is not really nessecary because the method would a NOP then:
```Java
   public static void setKeyPressed(InputUtil.Key key, boolean pressed) {
      KeyBinding keyBinding = (KeyBinding)keyToBindings.get(key);
      if (keyBinding != null) {
         keyBinding.setPressed(pressed);
      }

   }
   ```
But I figured that we are already cancelling/overwriting every other method in that class we can also cancel this one. (And improve performance a little bit)

Reply:
Well, I'd still prefer not to cancel. It won't happen a lot, but if we get around a single incompatibility, I'd still consider it a win 😄